src/Steam.package/STAuthentificationComponent.class/instance/renderOAuth2Providers.on..st

```diff
@@ -16,7 +12,7 @@ renderOAuth2Providers: aCallbackUrl on: html
 	html anchor
 		callback: [ self signOnUsingOAuth2Microsoft: aCallbackUrl ] ;
 		with: 'Microsoft'.
-		html text: ' | '.
+		html text: ' | '."
```

Comment:
Commented code should be removed! If you want to recover old code: 
 - commit often
 - then you can go back to older commits ;)